adding postgres and pgadmin dockerfile (#3)
esoviscode/database: README.md

# pgadmin4 and Postgres server

To launch docker containers with Postgres and pgadmin4 run command:
```bash
docker-compose up -d
```
Then open your browser and go to the url:
```

> localhost:5555

Reviewer: ten URL nie działa, raczej powinien być taki:
```suggestion
http://localhost:5555
```

Author: Normalnie u mnie działa

Author: Ale może być i tak

```
E-mail:
``` 
esovisco@pg.edu.pl 
```
Password
```
1qazaq1
 ```
 After succesfull logging click Servers-> Register -> Server

 ![image](https://cdn.discordapp.com/attachments/1044342494143451166/1053010730733027468/image.png)

Give the name of the  database.
To connect client with database you have to know the ip of database.

> You can find it typing docker inspect (id of docker conatiner) in the Terminal, as a field IPAddress. Go to connection tab and insetr

Reviewer: dokumenty zawijamy do 80 znaków

> You can find it typing docker inspect (id of docker conatiner) in the Terminal, as a field IPAddress. Go to connection tab and insetr

Reviewer: ```suggestion
You can find it by typing `docker inspect <id of docker container>` in the Terminal, as a field IPAddress. Go to connection tab and insert
```

Reviewer: Dodatkowo w README tego repozytorium jest komenda, która daje właśnie IP korzystając z inspecta

> You can find it by typing `docker inspect <id of docker container>` in the 

Reviewer: To było wcześniej w readme, myślę że to lepsza metoda na znalezienie adresu IP bazy
```suggestion
You can find it by typing `docker inspect  -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' postgresovisco`
```

Reviewer: @kapselccc
